inc/comments-template.php

```diff
@@ -9,11 +9,14 @@
  */
 
 $talk_url = get_option( 'coral_talk_base_url' );
+$static_url = get_option( 'coral_talk_static_url', $talk_url );
 $talk_container_classes = get_option( 'coral_talk_container_classes' );
 $div_id = 'coral_talk_' . absint( rand() );
 if ( ! empty( $talk_url ) ) : ?>
```

Comment:
Check that `$static_url` is not empty?

Reply:
The `static_url` fallback value is the value of `talk_url`, so it won't be empty if `talk_url` isn't.